doc/source/how-to/documenting.rst

```diff
@@ -0,0 +1,587 @@
+Documenting
+===========
+PyAnsys documentation must not only be written but also maintained. If you are
+new to writing developer documentation, see the `Google Developer Documentation
+Style Guide <https://developers.google.com/style>`_. It provides
+editorial guidelines for writing clear and consistent developer documentation,
+allowing this guide to supply guidance specific to PyAnsys library
+documentation.
+
+When writing developer documentation, the relationship between code and
+documentation is key. To keep documentation up to date with radial evolving
+code:
+
+- Minimize the content footprint
+- Write `timeless documentation <https://developers.google.com/style/timeless-documentation>`_
+- Support contributions from both inside and outside of the development team
+- Perform periodic reviews
```

Comment:
```suggestion
- Minimize the content footprint.
- Write `timeless documentation <https://developers.google.com/style/timeless-documentation>`_.
- Support contributions from both inside and outside of the development team.
- Perform periodic reviews.
```

Reply:
Periods ending each bullet because they are imperative sentences (complete sentences with an understood subject of "you".